src/actionmanager.cpp

```diff
@@ -9,9 +9,14 @@
 
 ActionManager::ActionManager(QObject *parent) : QObject(parent)
 {
+    isSaveRecentsEnabled = true;
```

Comment:
This didn't follow the usual pattern of initializing it to a default value. Not that it matters because we are loading the real settings right after, but it might as well be consistent with the rest of the code.